src/web3/KeychainAccountManager.ts

```diff
@@ -174,17 +164,21 @@ export async function clearStoredAccounts() {
 }
 
 /**
- * Implements the signer interface on top of the OS keychain
+ * Manages a single account (single address) in the keychain
```

Comment:
I'm maybe a little confused by terminology throughout this PR. "account" seems to refer to a single address, but seems to be a one-to-many mapping from address -> "wallets" where each wallet exists on its own chain. Is that correct? Since we can have multiple `KeychainAccountManager` instances within `KeychainWallet` this seems to _kind of_ set up some of the scaffolding for multi-account as well, right? (i.e., multi-address, multi-chain vs single-address, multi-chain).

Reply:
I'm also confused by the terminology. I know it's annoying to debate naming, but since it is relatively early in the project, it's probably the best time to align. My main point of confusion is: what is a wallet, and what is an account? KeychainWallet being a singleton at the top level makes me think the wallet is a stand-in for Valora as an application, which can manage multiple accounts, each of which has their own private key and account address. However, the "Wallet" class deals only with one private key and account address.

Reply:
I think the main point of confusion for me stems from the naming of KeychainWallet. I would probably flip the names KeychainWallet and KeychainAccountManager. "account manager" sounds like something that could manage multiple accounts, and from there on down, wallet and account are used interchangeably, which is a little annoying but not any worse than the rest of our codebase lol

Reply:
ok I just noticed that KeychainWallet was already named that before this PR, and inherits from other things called "wallet". kinda ruins my idea lol